harness/determined/common/api/request.py

```diff
@@ -84,288 +74,7 @@ def make_interactive_task_url(
-        headers=headers,
-        authenticated=authenticated,
-        auth=auth,
-        cert=cert,
-        timeout=timeout,
-    )
-
-
-def patch(
-    host: str,
-    path: str,
-    json: Dict[str, Any],
-    headers: Optional[Dict[str, str]] = None,
-    authenticated: bool = True,
-    auth: Optional[authentication.Authentication] = None,
-    cert: Optional[certs.Cert] = None,
-    timeout: Optional[Union[Tuple, float]] = None,
-) -> requests.Response:
-    """
-    Send a PATCH request to the remote API.
-    """
-    return do_request(
-        "PATCH",
-        host,
-        path,
-        json=json,
-        headers=headers,
-        authenticated=authenticated,
-        auth=auth,
-        cert=cert,
-        timeout=timeout,
-    )
-
-
-def put(
-    host: str,
-    path: str,
-    json: Optional[Dict[str, Any]] = None,
-    headers: Optional[Dict[str, str]] = None,
-    authenticated: bool = True,
-    auth: Optional[authentication.Authentication] = None,
-    cert: Optional[certs.Cert] = None,
-    timeout: Optional[Union[Tuple, float]] = None,
-) -> requests.Response:
-    """
-    Send a PUT request to the remote API.
-    """
-    return do_request(
-        "PUT",
-        host,
-        path,
-        json=json,
-        headers=headers,
-        authenticated=authenticated,
-        auth=auth,
-        cert=cert,
-        timeout=timeout,
-    )
-
-
 def browser_open(host: str, path: str) -> str:
     url = make_url(host, path)
     webbrowser.open(url)
     return url
-
-
-class WebSocket:
-    def __init__(self, socket: Any) -> None:
-        import lomond
-
-        self.socket = socket  # type: lomond.WebSocket
-
-    def __enter__(self) -> "WebSocket":
-        return self
-
-    def __iter__(self) -> Iterator[Any]:
-        from lomond import events
-
-        for event in self.socket.connect(ping_rate=0):
-            if isinstance(event, events.Connected):
-                # Ignore the initial connection event.
-                pass
-            elif isinstance(event, (events.Closing, events.Disconnected)):
-                # The socket was successfully closed so we just return.
-                return
-            elif isinstance(
-                event,
-                (events.ConnectFail, events.Rejected, events.ProtocolError),
-            ):
-                # Any unexpected failures raise the standard API exception.
-                raise errors.BadRequestException(message="WebSocket failure: {}".format(event))
-            elif isinstance(event, events.Text):
-                # All web socket connections are expected to be in a JSON
-                # format.
-                yield _json.loads(event.text)
-
-    def __exit__(
-        self,
-        exc_type: Optional[type],
-        exc_val: Optional[BaseException],
-        exc_tb: Optional[types.TracebackType],
-    ) -> None:
-        if not self.socket.is_closed:
-            self.socket.close()
-
-
-def ws(host: str, path: str) -> WebSocket:
```

Comment:
Was this just never used? My grepping powers don't show any evidence of it.

Reply:
"never" could refer to past history.  It was used once.  But it isn't in use anymore.

Reply:
Am good to delete. Resolving.